### PR TITLE
update image build scripts to handle buster

### PR DESCRIPTION
**Pull request by bchavet**

### CODE REVIEW

File: Dockerfile.apt.template [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90tugboat && \
     \
     apt-get install -y \
         git locales sudo openssh-client ca-certificates tar gzip build-essential \
-        net-tools netcat-openbsd unzip zip bzip2 gnupg curl wget vim ssmtp runit \
+        net-tools netcat-openbsd unzip zip bzip2 gnupg curl wget vim runit \
```
Comment (deviantintegral) [inline Dockerfile.apt.template]: Is mail collection handled without ssmtp now?

Do we have release notes somewhere so if users had ssmtp in their scripts they know they need to install it now?

File: services/php-apache/Dockerfile [hunk 2]
```diff
@@ -3,10 +3,11 @@ ENV COMPOSER_NO_INTERACTION=1 \
     COMPOSER_ALLOW_SUPERUSER=1 \
     DOCROOT=/var/www/html
 
-RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev mysql-client && \
+RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev && \
+    (apt-get install -y mysql-client || apt-get install -y mariadb-client) && \
```
Comment (deviantintegral) [inline services/php-apache/Dockerfile]: Interesting, why the or here?

File: services/php-apache/Dockerfile [hunk 3]
```diff
@@ -3,10 +3,11 @@ ENV COMPOSER_NO_INTERACTION=1 \
...
     \
-    docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ && \
-    docker-php-ext-install -j$(nproc) gd iconv pdo_mysql && \
+    (docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ || /bin/true) && \
+    (docker-php-ext-install -j$(nproc) gd iconv pdo_mysql || /bin/true) && \
```
Comment (deviantintegral) [inline services/php-apache/Dockerfile]: Don't we want to fail if these extensions fail to build?

File: services/php-apache/Dockerfile (hunk 2 above)
Comment (bchavet) [reply services/php-apache/Dockerfile]: buster doesn't have a `mysql-client` package, only `mariadb-client`

File: services/php-apache/Dockerfile (hunk 3 above)
Comment (bchavet) [reply services/php-apache/Dockerfile]: there's a problem with buster & php7.3 where these will fail to build, don't want that to block the other versions

File: Dockerfile.apt.template (hunk 1 above)
Comment (bchavet) [reply Dockerfile.apt.template]: https://github.com/Lullabot/tugboat/issues/3257

File: services/php-apache/Dockerfile (hunk 3 above)
Comment (bchavet) [reply services/php-apache/Dockerfile]: https://github.com/docker-library/php/issues/865

**Pull request merged**